dags/make_packet.py

```diff
@@ -45,6 +45,6 @@
             'AWS_SECRET_ACCESS_KEY': AWS_SECRET_ACCESS_KEY,
             'S3_BUCKET_NAME': S3_BUCKET_NAME,
         },
-        force_pull=True
+        force_pull=True,
+        auto_remove=True
```

Comment:
A teachable moment from me to you, @antidipyramid! We run our Airflow DAGs in containers. I wrote a special Docker operator that's [automatically configured to remove containers](https://github.com/datamade/la-metro-dashboard/blob/1dc2d26f918ed68d961d26d8a929604bbb7cdaae/operators/blackbox_docker_operator.py#L37), but I used [the standard Docker operator](https://airflow.apache.org/docs/apache-airflow-providers-docker/stable/_api/airflow/providers/docker/operators/docker/index.html) for this DAG and neglected to configure this option, meaning we had 100+ containers build up over time, eventually chewing through all the space on the server. 🙀